src/components/InputModal.vue

```diff
@@ -31,7 +29,7 @@ export default {
   props: {
     show: Boolean,
   },
-  emits: ["close"],
+  emits: [close],
```

Comment:
is this working?

Reply:
It is used for modal, and modal is working . but I have to check  properly whether it affects modal or not

Reply:
no i mean event should be a `string`